internal/extension/smartagentextension/factory.go

```diff
@@ -41,18 +42,24 @@ func NewFactory() component.ExtensionFactory {
 }
 
 var bundleDir = func() string {
-	out := os.Getenv(constants.BundleDirEnvVar)
-	if out == "" {
-		exePath, err := os.Executable()
-		if err != nil {
-			panic("Cannot determine agent executable path, cannot continue")
-		}
-		out, err = filepath.Abs(filepath.Join(filepath.Dir(exePath), ".."))
-		if err != nil {
-			panic("Cannot determine absolute path of executable parent dir " + exePath)
+	dir := os.Getenv(constants.BundleDirEnvVar)
+	if dir == "" {
+		if runtime.GOOS == "windows" {
+			dir = filepath.Join("C:\\", "Program Files", "Splunk", "OpenTelemetry Collector", "agent-bundle")
```

Comment:
```suggestion
			dir = filepath.Join("C:", "Program Files", "Splunk", "OpenTelemetry Collector", "agent-bundle")
```
I am not familiar with Windows -- will this work?

Reply:
I think `C:relative_path` and `C:\\absolute_path` in windows, but I'm having trouble getting the desired absolute value w/ Join.

Reply:
`filepath.Join("C:", "Program Files")` seems to work for me on a Windows VM -- it returns an absolute path and I can take a directory listing from it 🤷

Reply:
Weird, the windows-test unit test run failed with that and `"C:\\", "Program Files"` but `"C:\\Program Files", ...` works

Reply:
I think you'll want to `os.Getenv("programfiles")` since the default location can vary.